Generics_lesson17/Task2/Orange.java

```diff
@@ -0,0 +1,9 @@
+package Generics_lesson17.Task2;
+
+public class Orange extends Fruit {
+    private Float mass = 1.5F;
```

Comment:
Это неизменное значение, стало быть константа.
```suggestion
    private static final Float MASS = 1.5F;
```